2Dgame/Assets/Scripts/PlatformSpawnerScript.cs

```diff
@@ -40,83 +49,75 @@ public class PlatformSpawnerScript : MonoBehaviour {
 	 * 		10% chance trampoline
 	 * 		10% chance moving
 	 */ 
-	GameObject basicPlatformScheme() {
-        return PlatformScheme(80, 10, 10);
+	void basicPlatformScheme() {
+        PlatformScheme(80, 10, 10);
 	}
 
     /**
      * Platformscheme with a variable amount of parameters.
      * 
      * Returns a platform prefab.
      */
-    GameObject PlatformScheme(params int[] percentages)
+    void PlatformScheme(params int[] percentages)
     {
-        if (didWeFlip)
-        {
-            flipCounter++;
-            if (flipCounter % 6 == 0)
-            {
-                didWeFlip = false;
-                return platformPrefabs[3];
-            }
-        }
-
         int randomInt = Random.Range(0, 100);
         int percentageCounter = 0;
         for (int i = 0; i < percentages.Length; i++)
         {
             percentageCounter += percentages[i];
             if (randomInt < percentageCounter)
             {
+                prefabqueue.Enqueue(platformPrefabs[i]);
                 if (i == 3)
                 {
-                    if (didWeFlip)
+                    for (int j = 0; j < 3; j++)
                     {
-                        flipCounter = 0;
+                        prefabqueue.Enqueue(platformPrefabs[0]);
                     }
-                    else
+                    prefabqueue.Enqueue(platformPrefabs[3]);
+                }
+                else if (i == 4)
+                {
+                    for (int j = 0; j < 3; j++)
                     {
-                        flipCounter++;
+                        prefabqueue.Enqueue(platformPrefabs[5]);
                     }
-					didWeFlip = !didWeFlip;
                 }
-
-                return platformPrefabs[i];
+                return;
             }
         }
-        return platformPrefabs[0];
     }
 
     /**
      * Method for choosing platformscheme.
      * Is the game's primary progressionsystem.
      * Difficulty increases as score increases.
      */
-    GameObject choosePlatform()
+    void choosePlatform()
     {
         if (this.transform.position.y < 200)
         {
-            return basicPlatformScheme();
+            basicPlatformScheme();
         }
         else if (this.transform.position.y < 500)
         {
-            return PlatformScheme(45, 45, 10);
+            PlatformScheme(45, 45, 10);
         }
         else if (this.transform.position.y < 700)
         {
-            return PlatformScheme(0, 100); // First all moving
+            PlatformScheme(0, 100); // First all moving
         }
         else if (this.transform.position.y < 1000)
         {
-            return PlatformScheme(60, 20, 10, 10); // Flips
+            PlatformScheme(60, 20, 10, 10); // Flips
         }
         else if (this.transform.position.y < 1500)
         {
-            return PlatformScheme(45, 40, 5, 10); // More hard stuff
+            PlatformScheme(30, 40, 5, 15, 10); // More hard stuff, first blink
```

Comment:
we should probably decide together exactly how the progression should be, maybe when we finalized the game for release. I'll leave it right now.